dhall-csv/src/Dhall/Csv/Util.hs

```diff
@@ -0,0 +1,16 @@
+module Dhall.Csv.Util (encodeCsvDefault) where
+
+import Data.Text (Text)
+
+import qualified Data.ByteString.Lazy   as ByteString
+import qualified Data.Csv
+import qualified Data.HashMap.Strict    as HashMap
+import qualified Data.Vector            as Vector
+import qualified Data.Text.Encoding
+
+encodeCsvDefault :: [Data.Csv.NamedRecord] -> Text
+encodeCsvDefault csv = Data.Text.Encoding.decodeUtf8 $ ByteString.toStrict $ Data.Csv.encodeByName header csv
+  where
+    header = case csv of
+        [] -> Vector.empty
+        (m:_) -> Vector.fromList $ HashMap.keys m
```

Comment:
The output ordering of `keys` is arbitrary and depends on the `hashable` version and platform. Maybe `sort` the keys?

Reply:
Thanks! I didn't know that, I thought it already had them sorted. I'll add the a `sort` for now, but I'm thinking it may be a good idea to return a header too in the `dhallToCsv` function, I'll try to do that in a future PR